$NetBSD: patch-Makefile,v 1.7 2014/08/10 20:51:01 wiz Exp $

Disable apropos, makewhatis, and man.cgi until mdocml includes a
portable fts implementation.

--- Makefile.orig	2014-08-10 17:26:46.000000000 +0000
+++ Makefile
@@ -57,7 +57,7 @@ INSTALL_MAN	 = $(INSTALL_DATA)
 # Be careful: the fts(3) implementation in glibc is broken on 32bit
 # machines, see: https://sourceware.org/bugzilla/show_bug.cgi?id=15838
 #
-BUILD_TARGETS	+= db-build
+#BUILD_TARGETS	+= db-build
 
 # The remaining settings in this section
 # are only relevant if db-build is enabled.
